src/utils/AccessibleButton.ts

```diff
@@ -151,12 +151,18 @@ export class AccessibleButton {
       this.bindEvent(
         'keydown',
         KeyboardUtils.keypressAction(KEYBOARD.SPACEBAR, (e: Event) => {
+          if (e.target instanceof HTMLInputElement) {
+            return;
+          }
```

Comment:
The reason spaces couldn't be entered in `CategoryFacetSearch` was because it's a descendant of an `AccessibleButton`.

Although Firefox is reporting this event as being a bubbling event, it behaved like a capturing event and always ran prior to events directly bound on a descending input.

Reply:
🙏